src/views/Header.js

```diff
@@ -34,9 +34,9 @@ type SubViewProps = NavigationSceneRendererProps & {
   onNavigateBack?: () => void,
 };
 
-type Navigation = NavigationScreenProp<NavigationState, NavigationAction>;
+type Navigation = NavigationScreenProp<*, NavigationAction>;
 
-type SubViewRenderer = (subViewProps: SubViewProps) => ?React.Element<*>;
+type SubViewRenderer = (subViewProps: SubViewProps) => ?React.Element<any>;
```

Comment:
I might be not the best one in Flow, but my understanding was that with `*`, it was inferring the type from first renderSubView method (renderLeftComponent) and so, it was complaining that the renderTitleComponent return value is different than in renderLeftComponent (saying prop types don't match). I guess it's because we didn't tell it that renderSubView can return *any*thing?